src/components/Header.js

```diff
@@ -20,13 +20,17 @@ class Header extends React.Component {
 
   constructor(props) {
     super(props);
-    const { description, image, twitterHandle } = props;
+    const { description, image, twitterHandle, title } = props;
     const meta = {
       'twitter:site': 'opencollect',
       'twitter:creator': twitterHandle,
       'fb:app_id': '266835577107099',
       'og:image': image,
       description: truncate(description, 256),
+      'twitter:card': 'summary',
+      'twitter:title': `${title} - Open Collective`,
```

Comment:
It's only when `title` is defined that we should add it to that string.

```js
const meta = {
...
  'twitter:title': 'Open Collective',
...
}
if (title) {
  meta['twitter:title'] = `${title} - Open Collective`
}
```

Reply:
Alright. will fix.